### PR TITLE
[RW-5195][risk = no] fill in descriptions for columns

### PR DESCRIPTION
**Pull request by jaycarlton**
Populate column descriptions in the BigQuery Schema and the Swagger input. It should always be possible to update existing BigQuery  tables with new column descriptions.

Introduce a new input file that's just a yaml map from column name to description. At some point I want to simplify and consolidate input files, but for now, it's all right.

One more  output format to give an unpopulated definitions yaml file for input.

In the course of researching the cohort table columns, [I realized](https://pmi-engteam.slack.com/archives/C0189PPPKFF/p1601482610014800) I needed to drop a couple of columns that aren't user-facingi or for external consumption (at least not yet).

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/reporting/schemas/input/descriptions/user.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+user:
+  about_you: User's description of themselves.
```
Comment (NehaBroad) [inline api/reporting/schemas/input/descriptions/user.yaml:3]: about_you should be like User's description of their research background, experience and research interests.

File: api/reporting/schemas/input/descriptions/user.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  area_of_research: 
+  compliance_training_bypass_time: |
+        Time Compliance Training administratively bypassed, or null if this requirement
+        was never bypassed for this user.
+  compliance_training_completion_time: |
+    Time Compliance Training completed, or null if never completed.
+  compliance_training_expiration_time: |
+      Time  User's compliance training will expire, or null training not taken or if bypassed
```
Comment (NehaBroad) [inline api/reporting/schemas/input/descriptions/user.yaml:11]: should there be if between null and training?

File: api/reporting/schemas/input/descriptions/user.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    Version of DUCC last signed by this user.
+  demographic_survey_completion_time: |
+    Timestamp for completion of Demographic Survey (if completed).
+  disabled: |
+    If true, this account has been disabled by an administrator (or potentially an automatic process).
+  era_commons_bypass_time: |
+    Time an administrator bypassed the ERA Commons requirement for this user.
+  era_commons_completion_time: |
+    Time user completed ERA Commons account link.
+  family_name: |
```
Comment (NehaBroad) [inline api/reporting/schemas/input/descriptions/user.yaml:33]: nit: can we put family name next to given_name and username?

File: api/reporting/schemas/input/descriptions/user.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    Override value for the default free tier time limit (days).
+  free_tier_credits_limit_dollars_override: |
+    Override value for the default free tier spending limit (USD).
+  given_name: |
+    User first name (given name).
+  last_modified_time: |
+    Time of last modification to this user account.
+  professional_url: |
+    User's URL at primary place of work.
+  two_factor_auth_bypass_time:  |
```
Comment (NehaBroad) [inline api/reporting/schemas/input/descriptions/user.yaml:49]: nit: can we put all bypass_time elements and completion_time together?

File: api/reporting/schemas/input/descriptions/workspace.yaml [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    tier project, this indicates whether a user has an available balance in their quota.  For a
+    user-provided billing account, this corresponds to whether payment is valid and up to date.
+  cdr_version_id: |
+      Foreign key into CDR table.
+  creation_time: |
+      Time workspace was initially created.
+  creator_id: |
+      User ID of user who initially created this workspace.
+  disseminate_research_other: |
+      Description of user-defined research dissemination option (when "Other" is selected).
```
Comment (NehaBroad) [inline api/reporting/schemas/input/descriptions/workspace.yaml:16]: i think we should remove the brackets because this is the description of other

File: api/reporting/schemas/input/descriptions/workspace.yaml [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+  last_accessed_time: |
+      No longer in use. Column should be ignored.
+  last_modified_time: |
+      Last time a modification was made to this workspace, with  certain exceptions. In general,
+      only changes that should alter sort order in the Recent Workspaces UI trigger an update
+      to the last modified time.
+  name: |
+      User-defined name for this workspace. Human-readable.
+  needs_rp_review_prompt: ''
```
Comment (NehaBroad) [inline api/reporting/schemas/input/descriptions/workspace.yaml:25]: If yes then ask the owner of the workspace to review the Research purpose

File: api/src/main/java/org/pmiops/workbench/db/dao/CohortDao.java [hunk 7]
```diff
@@ -37,8 +37,6 @@
           + "  c.description,\n"
           + "  c.lastModifiedTime,\n"
           + "  c.name,\n"
-          + "  c.type,\n"
```
Comment (NehaBroad) [inline api/src/main/java/org/pmiops/workbench/db/dao/CohortDao.java:40]: Why did you remove type and version?

Comment (NehaBroad) [reply api/src/main/java/org/pmiops/workbench/db/dao/CohortDao.java:40]: OK so just re-read the conversation from data-apps channel

**Review by NehaBroad — APPROVED** [review]
Just few nits and good to merge

File: api/reporting/schemas/input/descriptions/user.yaml (hunk 3 above)
Comment (jaycarlton) [reply api/reporting/schemas/input/descriptions/user.yaml:33]: This file doesn't determine the order, but for my own sanity I  alphabetized all the columns except for  snapshot_timestamp.

You might be interested in the [new formatted query output](https://github.com/all-of-us/workbench/pull/4108), though, which adds structure to the huge user   and workspace  rows.

File: api/reporting/schemas/input/descriptions/user.yaml (hunk 4 above)
Comment (jaycarlton) [reply api/reporting/schemas/input/descriptions/user.yaml:49]: no, I need to  keep  it alphabetical  for  merging, tracking across  files, etc.
